Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -116,6 +116,14 @@ void Processor::flush_tlb_local(VirtualAddress, size_t)
     asm volatile("isb");
 }
 
+void Processor::flush_entire_tlb_local()
+{
+    asm volatile("dsb ishst");
+    asm volatile("tlbi vmalle1is");
+    asm volatile("dsb ish");
+    asm volatile("isb");
```

Comment:
thereʼs `flush` method in `ASM_wrapper` that you could use here for more clarity(?). or just remove that from here and invoke it on call-site?

Reply:
I think for now this is fine. To me the `flush` method doesn't really describe what it is actually flushing anyway.

Reply:
troo!